docs/source/subcommands.md

```diff
@@ -38,6 +38,24 @@ $ latch register PATH_TO_WORKFLOW_DIRECTORY
 
 The first argument specifies the local path in which to look for workflow objects. Inside the local path should be a single directory called `wf` containing `__init__.py`, any helper python files, a `Dockerfile`, and a `version` files. See [here](basics/what_is_a_workflow.md) for more info.
 
+### Options
+
+#### `--disable-auto-version`, `-d`
+
+Whether to automatically bump the version of the workflow each time register is called.
```

Comment:
```suggestion
Do not include the workflow contents hash in the workflow version. The `version` must be manually updated after each registration.
```
+ explain why this might be useful

Reply:
or delete if not useful lol